microsoft-edge/dev-videos/index.md

```diff
@@ -30,265 +30,816 @@ To add a new video:
 
-To learn more about the CSS scroll-linked animations feature, see [@scroll-timeline](https://developer.mozilla.org/docs/Web/CSS/@scroll-timeline) at MDN.
+See also:
+* [View formatted JSON](../devtools-guide-chromium/json-viewer/json-viewer.md)
 
 
 <!-- ====================================================================== -->
-## DevTools - Customizing Microsoft Edge Developer Tools and quick feature access
+## Web Platform - Fully style the drop-down part of a `<select>` with the new `<selectmenu>` element
 
-_Published on 5 May 2022._
+May 31, 2022
 
-[![Thumbnail image for the DevTools customization video](./index-images/customize-devtools.png)](https://www.youtube.com/watch?v=ypRzEBYNptQ)
+[![Thumbnail image for the selectmenu video](./index-images/selectmenu.png)](https://www.youtube.com/watch?v=Ts7jvRyQACY)
 
-Learn how to customize DevTools to fit your needs.
+Styling `<select>` elements has been challenging. The experimental<!-- todo: update.  June 2024: the present repo & MDN don't mention selectmenu --> `<selectmenu>` element promises to overcome the remaining limitations by enabling web developers to style all parts of the element.
 
-This video covers how to dock or undock DevTools, open new tools, and close those you don't need. It describes how to move tools in the bottom drawer and customize the text size and theme. The video also explains how you can use the Command Menu keyboard shortcuts to quickly customize DevTools.
+See also:
+* [Styling `<select>` elements for real](https://blogs.windows.com/msedgedev/2022/05/05/styling-select-elements-for-real/) - blog post about styling `<select>` elements and the `<selectmenu>` element.
 
 
 <!-- ====================================================================== -->
-## DevTools - What's New in DevTools 101
+## Web Platform - Create a scroll-linked animation without JavaScript
 
-_Published on 28 Apr 2022._
+May 12, 2022
 
-[![Thumbnail image for the DevTools What's New in 101 video](./index-images/devtools-whatsnew-101.png)](https://www.youtube.com/watch?v=kv6Q8a9bsbA)
+[![Thumbnail image for the scroll-linked animation API video](./index-images/scroll-linked-animations.png)](https://www.youtube.com/watch?v=Q0nhiHVVnvI)
+
+Learn about the upcoming CSS scroll-linked animations feature and how it can be used to create a reading progress indicator on a webpage, without using JavaScript.
 
-Learn more about our latest announcements from the Microsoft Edge DevTools team at [What's New in DevTools 101](../devtools-guide-chromium/whats-new/2022/04/devtools-101.md).
+CSS scroll-linked animations is an experimental feature<!-- todo: update.  repo doesn't mention scroll-linked --> in Microsoft Edge.  To try this feature, go to `edge://flags` and then enable the **Experimental Web Platform features** setting.<!-- exists as of June 2024 -->
```

Comment:
CSS scroll-linked animations was experimental feature, how to word?
@captainbrosset

Reply:
differentiated 2022 vs 2023 situation, and two See Alsos.